README.md

```diff
@@ -29,7 +29,7 @@ class Project < ApplicationRecord
 
   validates :title, presence: true
 
-  validates :preview, attached: true
+  validates :preview, attached: true, content_size: { less_than: 100.megabytes , message: 'is not given between size' }
```

Comment:
maybe rename content_size to "size" or "file_size"
since it's more about file

Reply:
@igorkasyanchuk request to change it to active_storage_size prevent conflict with other 3party gems. please suggest.

Reply:
this is for "size" and "file_size"? who is using such validations? Maybe file_size is available?

Reply:
@reckerswartz do you have any other comments? this PR looks like 95% ready, if you can make fixes I'll merge it.

So I see 2 things which I would like to change:
- name of validator (find a better name if possible)
- use number_to_human_size (see my other comment in PR)

Reply:
@igorkasyanchuk sorry for delay!
updated as per requested but still test are pending :smile: